### PR TITLE
feat: update figma plugin UI

### PR DESCRIPTION
**Pull request by nicholasrice**
# Description
Updates UI to match design spec.

## Motivation & context
This work primarily update the Figma plugin UI to match design spec. It also adds the ability to customize the accent-color for a node tree. It also begins the migration to web-components. At the time this was originally authored there were several bugs that prevented whole-sale conversion to web components so the primary view is still in React, but pieces have been moved over.

This change is currently published to the Figma plugin registry for demo purposes so I want to avoid substantial changes.

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [x] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: packages/fast-figma-plugin-msft/src/core/ui/components/stealth-button/index.ts [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+})
+@designSystemConsumer({
+    recipes: [
+        neutralFillStealthRest,
+        neutralFillStealthHover,
+        neutralFillStealthActive,
+        accentForegroundCutRest,
+    ],
+})
+export class StealthButton extends FastElement {
```
Comment (janechu) [inline packages/fast-figma-plugin-msft/src/core/ui/components/stealth-button/index.ts:74]: would this be deprecated in the future to use a stealth button from fast-components in the future?

File: packages/fast-figma-plugin-msft/src/core/ui/components/swatch/index.ts [hunk 2]
```diff
@@ -0,0 +1,143 @@
+import { attr, css, customElement, FastElement, html } from "@microsoft/fast-element";
+import { designSystemConsumer } from "@microsoft/fast-components/dist/design-system-consumer";
```
Comment (janechu) [inline packages/fast-figma-plugin-msft/src/core/ui/components/swatch/index.ts:2]: seems like this should be exported from the root

File: packages/fast-figma-plugin-msft/src/core/ui/index.tsx [hunk 3]
```diff
@@ -10,9 +10,21 @@ import {
 import React from "react";
 import { MessageAction, MessageTypes, UIMessage } from "../messaging";
 import { RecipeData, RecipeTypes } from "../recipe-registry";
-import Swatch from "./swatch";
-import { DesignSystem, StandardLuminance } from "@microsoft/fast-components-styles-msft";
-import { refresh, revertChanges } from "./glyphs";
+// import Swatch from "./swatch";
```
Comment (janechu) [inline packages/fast-figma-plugin-msft/src/core/ui/index.tsx:13]: should we remove this?

File: packages/fast-figma-plugin-msft/src/core/ui/index.tsx [hunk 4]
```diff
@@ -10,9 +10,21 @@ import {
...
+import {
+    DesignSystem,
+    StandardLuminance,
+    DesignSystemDefaults,
+} from "@microsoft/fast-components-styles-msft";
+import { refresh, revertChanges, detach } from "./glyphs";
+import { Drawer, Swatch, CornerRadius } from "./components";
+import { ColorRGBA64, parseColorHexRGB } from "@microsoft/fast-colors";
+
+/* tslint:disable:no-unused-expression */
```
Comment (janechu) [inline packages/fast-figma-plugin-msft/src/core/ui/index.tsx:23]: is tslint still being used in this package?

File: packages/fast-figma-plugin-msft/src/ui.tsx [hunk 5]
```diff
@@ -2,7 +2,12 @@ import React from "react";
 import ReactDOM from "react-dom";
 import { PluginUI, PluginUIProps } from "./core/ui";
 import { UIMessage } from "./core/messaging";
+import { FASTDesignSystemProvider } from "@microsoft/fast-components";
 
+// tslint:disable
```
Comment (janechu) [inline packages/fast-figma-plugin-msft/src/ui.tsx:7]: don't think you need these anymore

File: packages/fast-figma-plugin-msft/src/core/ui/components/swatch/index.ts (hunk 2 above)
Comment (nicholasrice) [reply packages/fast-figma-plugin-msft/src/core/ui/components/swatch/index.ts:2]: The API is going to change on it, so it's private for the time being.

File: packages/fast-figma-plugin-msft/src/core/ui/components/stealth-button/index.ts (hunk 1 above)
Comment (nicholasrice) [reply packages/fast-figma-plugin-msft/src/core/ui/components/stealth-button/index.ts:74]: Yes

**Pull request merged**